package.json

```diff
@@ -45,7 +47,7 @@
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-qunit": "^4.6.0",
     "ember-resolver": "^7.0.0",
-    "ember-source": "~3.16.0",
+    "ember-source": "~3.4.8",
```

Comment:
This seems somewhat odd, why drop this down to 3.4?

Reply:
Oh, good catch! Don't remember exactly, been a while. But I guess I just downgraded locally to investigate something failing, and accidentally commited that. Just reverted that change!